From b09bdfdaeebf409e82685344cbc32ecb473c9069 Mon Sep 17 00:00:00 2001
From: Nils Asmussen <nils@os.inf.tu-dresden.de>
Date: Fri, 14 Sep 2012 15:19:21 +0200
Subject: [PATCH] The aux field of Hip_mem should be an mword. This gives NOVA
 more flexiblity and leads to a simpler userspace, because
 you can put in an arbitrary pointer.

---
 include/hip.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hip.h b/include/hip.h
index cce2c43..fd446e0 100644
--- a/include/hip.h
+++ b/include/hip.h
@@ -45,7 +45,7 @@ class Hip_mem
         uint64  addr;
         uint64  size;
         uint32  type;
-        uint32  aux;
+        mword   aux;
 };
 
 class Hip
-- 
1.7.9.5

